Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -15,13 +17,14 @@ function defineListeners(cy, path, unixName) {
 		var node = e.cyTarget;
 		conceptDescription(path, node);
 		getTexts(node, path, unixName);
+		getEntireText(node,path);
```

Comment:
Understood. Removed the "getEntireText" function call from defineListeners function.